spec/controllers/evaluations_controller_spec.rb

```diff
@@ -40,4 +40,57 @@
       expect(own_evaluation.positive_feedback_count).not_to eq(positive_feedback_count + 1)
     end
   end
+
+  describe 'GET export' do
+    let!(:mooc_provider) { MoocProvider.create(name: 'openHPI', logo_id: 'logo_openHPI.svg', url: 'https://example.com', api_support_state: :naive) }
+    let!(:other_mooc_provider) { MoocProvider.create(name: 'openSAP', logo_id: 'logo_openSAP.svg', url: 'https://example.com', api_support_state: :naive) }
+    let!(:course) { FactoryGirl.create(:course, mooc_provider: mooc_provider) }
+    let!(:evaluation) { FactoryGirl.create(:full_evaluation, course: course, rating: 5) }
+    let!(:evaluation2) { FactoryGirl.create(:full_evaluation, course: course, rating: 10) }
+    let!(:course2) { FactoryGirl.create(:course, mooc_provider: mooc_provider) }
+    let!(:evaluation3) { FactoryGirl.create(:full_evaluation, course: course2, rating: 2) }
+    let!(:other_course) { FactoryGirl.create(:course, mooc_provider: other_mooc_provider) }
+    let!(:evaluation4) { FactoryGirl.create(:full_evaluation, course: other_course, rating: 4) }
+
+    render_views
+
+    it 'returns all evaluations for specific course and specific provider' do
+      get :export, format: :json, provider: mooc_provider.name, course_id: course.provider_course_id
+      expect(JSON.parse(response.body)['evaluations'].first['user_evaluations'].first['rating']).to eql evaluation.rating
```

Comment:
Use subjects to reduce code duplication, e.g.:

`subject { JSON.parse(response.body)['evaluations'].first['user_evaluations'] }`
and
`subject.first['rating']`